Weather App/Common/AppSettings.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AppSettings.swift
+//  Weather App
+//
+//  Created by Bruno Benčević on 9/8/21.
+//
+
+import Foundation
+
+class AppSettings {
+    private static let SETTINGS_KEY = "settingsData"
```

Comment:
Where did you see properties being named in Swift like that? Is this the "Swifty" way?

Reply:
Since it is a static constant, the usual convention in other programming languages is to use upper snake-case. Changed to camel-case.